src/main.js

```diff
@@ -284,12 +304,7 @@ class MiniGraphCard extends LitElement {
           @click=${e => this.handlePopup(e, this.entity[id])}
           style=${entityConfig.state_adaptive_color ? `color: ${this.computeColor(state, id)};` : ''}>
           ${entityConfig.show_indicator ? this.renderIndicator(state, id) : ''}
-          <span class="state__value ellipsis">
-            ${this.computeState(isPrimary && tooltipValue || state)}
-          </span>
-          <span class="state__uom ellipsis">
-            ${this.computeUom(isPrimary && entity || id)}
-          </span>
+          ${this.unitFirstOrLast(isPrimary, tooltipValue, state, id, entity)}
```

Comment:
Please use the css-based approach suggested by @ildar170975  (in the card-mod) instead, to reduce code complexity (improving maintainability)

Thanks for the contribution!